source/includes/getting-started/_concepts.html.md

```diff
@@ -33,7 +33,7 @@ This image displays all of the components of Bonsai's Platform. The Platform ena
 * **SDK & Libraries:** enables you to connect your own simulators to the Bonsai AI Engine. They also enable you to integrate your application with your BRAIN.
 * **HTTP API:** enables AI creators and AI users to interact with the Bonsai AI Engine.
 
-After your BRAIN is created and trained, you can use it with your app. Ultimately, our platform enables you to build AI agents that work (or play) for you without a deep knowledge of machine learning.
+After your BRAIN is created and trained, you can use it with your app by connecting your app to the API generated by the AI Engine using the Bonsai library specific to your app's programming language. Ultimately, our platform enables you to build AI agents that work (or play) for you without a deep knowledge of machine learning.
```

Comment:
in my mind, 'app' == mobile.  i like application in this context since it encompasses everything.